R/symbol_list.R

```diff
@@ -10,7 +10,7 @@
 #'
 #' @noRd
 #' @return A named character vector.
-symbol_list <- function(..., .character_only = FALSE, .all=FALSE)
+symbol_list <- function(..., .character_only = FALSE, .all=FALSE, .S3=FALSE)
```

Comment:
Why is this necessary? I can't see that .S3 is used in this function?

Reply:
Seems like earlier I had some problems with `...` gobling up the `.S3` parameter. But that is not an issue anymore.